### ISSUE TITLE
sklearn.linear_model._coordinate_descent._alpha_grid does not have a normalize argument anymore

### ISSUE THREAD

**Issue by JohannesWiesner** [richford/groupyr#80]
When running:

```python
estimator = SGLCV(groups=groups
                  n_jobs=n_jobs,
                  random_state=rng,
                  suppress_solver_warnings=False,
                  l1_ratio=1,
                  normalize=True
                  )

estimator.fit(X,y)
```
I get the following error:

```TypeError: _alpha_grid() got an unexpected keyword argument 'normalize'```

I search for recent commits in sklearn, and with this [commit](https://github.com/scikit-learn/scikit-learn/commit/e41753ebd57c44ae91b389f190c43ddc0b384a75#diff-e57008681bd9073bb36410075b059daf3b306428b8e711e2b074fc56e712d220), `sklearn.linear_model._coordinate_descent._alpha_grid` does not accept the `normalize` parameter anymore.

**Comment by JohannesWiesner** [comment]
As written in the deprecation warnings with scikit-learn > 1.2:

> ``normalize`` was deprecated in version 1.0 and will be removed in 1.2.

So with current version of groupyr would have to stick to scikit-learn >1.2

**Comment by arokem** [comment]
Thanks for raising this. I am working on this (and other dependency issues here: https://github.com/richford/groupyr/pull/79).

**Comment by richford** [comment]
I believe this as well was resolved with #79 but we just never got around to closing the issue. Can you try with v0.3.2? Please feel free to reopen if I'm mistaken.
